docusaurus/static/recipes-assets/splash-screen/SplashScreen.module.css

```diff
@@ -0,0 +1,41 @@
+@import "../../styles/imports/variables";
+
+:root {
+    --z-index-max: 2147483647;
+    --color-background: #221240;
+    --color-text: #fff;
+    --color-progress-bar: #f64f51;
+}
+
+.splashScreen {
+    position: fixed;
```

Comment:
can you put the mixin position to encourage our team to use it? 😅

Reply:
Is it `@position fixed, 0, 0, 0, 0` right?

Reply:
right! :)